packages/cli/templates/oauth-error-callback.handlebars

```diff
@@ -0,0 +1,19 @@
+<html>
+	<head>
+		<title>n8n - OAuth Callback</title>
+		<style>
+			body { font-family: 'Open Sans', sans-serif; padding: 10px;}
+			pre.error { background: #f7f7f7; border: 1px solid #ddd; border-radius: 3px; padding: 10px; overflow: auto; overflow-wrap: break-word; white-space: pre-wrap; }
+		</style>
+	</head>
+	<body>
+		Failed to connect. The window can be closed now.
```

Comment:
I'd recommend moving the sentence `The window can be closed now` to the bottom (after the `<pre>` block) and add a note stating that the error is outlined above, the user can use this information to fix the issue and retry.

Reply:
@krynble I have changed the position. thanks

Reply:
Hello @krynble  @netroy 
Is there anything else you think could be done here? if not, Can you please approve the PR?
Thank you.